sdk/python/tests/test_historical_retrieval.py

```diff
@@ -547,7 +588,26 @@ def test_historical_features_from_bigquery_sources(
             .reset_index(drop=True),
             check_dtype=False,
         )
+
         table_from_df_entities = job_from_df.to_arrow()
         assert_frame_equal(
             actual_df_from_df_entities, table_from_df_entities.to_pandas()
         )
+
+
+@pytest.mark.integration
```

Comment:
Is there a reason this is an integration test?

Reply:
Not anymore, will make that change.